src/temp/data.json

```diff
@@ -1,220 +1,221 @@
 {
-  "data": [
-    {
-      "id": 1,
-      "brand": "Nike",
-      "name": "Hiking BagPack",
-      "category": "Bagpack",
-      "details": "Etiam vehicula, sapien eget congue posuere, nisi diam mattis urna, in semper turpis nunc in odio. Aenean aliquet metus nec felis consequat luctus. Cras molestie sit amet sem sed fermentum",
-      "stock": 2,
-      "original_price": 2000,
-      "discount": 5,
-      "rating": 2,
-      "url": "https://cdn.shopify.com/s/files/1/1628/5651/products/product-22_large.png?v=1480682835"
-    },
-    {
+    "products": [
+      {
+        "id": 1,
+        "brand": "Nike",
+        "name": "Hiking BagPack",
+        "category": "BagPack",
+        "details": "Etiam vehicula, sapien eget congue posuere, nisi diam mattis urna, in semper turpis nunc in odio. Aenean aliquet metus nec felis consequat luctus. Cras molestie sit amet sem sed fermentum",
+        "original_price": 2000,
+        "discount_percent": 10,
+        "inStock": true,
+        "rating": 2,
+        "image_url": "https://cdn.shopify.com/s/files/1/1628/5651/products/product-22_large.png?v=1480682835"
+      },
+      {
         "id": 2,
         "brand": "Puma",
         "name": "Dome Tent",
         "category": "Tent",
         "details": "Etiam vehicula, sapien eget congue posuere, nisi diam mattis urna, in semper turpis nunc in odio. Aenean aliquet metus nec felis consequat luctus. Cras molestie sit amet sem sed fermentum",
-        "stock": 2,
         "original_price": 1000,
-        "discount": 5,
-        "rating": 2,
-        "url": "https://cdn.shopify.com/s/files/1/1628/5651/produc…06-497c-989f-ca0fde413b40_grande.jpg?v=1480931800"
+        "discount_percent": 5,
+        "inStock": true,
```

Comment:
A small nitpick...be consistent in your property naming...either use camelCase or under_score. Stick with one thing.

Reply:
i will change it .